metadata.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "simp-ds389",
+  "version": "0.1.0",
+  "author": "SIMP Team",
+  "summary": "Management of 389 Directory Server",
+  "license": "Apache-2.0",
+  "source": "https://github.com/simp/pupmod-simp-ds389",
+  "dependencies": [
+    {
+      "name": "simp/simplib",
+      "version_requirement": ">= 4.4.1 < 5.0.0"
+    },
+    {
+      "name": "puppetlabs/stdlib",
+      "version_requirement": ">= 6.3.0 < 7.0.0"
+    },
+    {
+      "name": "puppetlabs/concat",
+      "version_requirement": ">= 4.0.0 < 7.0.0"
+    }
+  ],
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "OracleLinux",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "7",
+        "8"
```

Comment:
Our manifest says we support EL8, but our acceptance tests don't test it.

I don't think it works, because `manifests/install.pp` doesn't accommodate the [EL8.1+ method of installing 389ds][0],which uses `yum module`.  It also no longer provides any of the packages in `ds389::install::$admin_package_list`.

[0]: https://directory.fedoraproject.org/docs/389ds/download.html#centos-81-ds-14x